### ISSUE TITLE
Invalid value for OmadaPortStatus.link_status

### ISSUE THREAD

**Issue by odouville** [MarkGodwin/tplink-omada-api#16]
(Seen on version 1.2.4)

When retrieving port status on my switch, I get the error "ValueError: 1 is not a valid LinkStatus" (see traceback below) on the first port whose link is up (tested on several switches, each raising the error on the first UP link).

The following code is used :
```python
    for port in switch.ports:
        print(f'Port {port.port}: name={port.name}, link={port.port_status.link_status.name}')
```

Accessing the property `port.port_status.link_status` without trying to get its `name` causes the same error.

I'll try to submit a pull request myself (could take some time as this is my first participation on github) as I know how to correct that.
However I am wondering if this is a matter of API version or something else ? Still not sure if modifying the LinkStatus enum is the correct fix to this issue, or if several versions of the Omada API should be managed.
For information, here is my api/info content :
```json
{
   "errorCode":0,
   "msg":"Success.",
   "result":{
      "controllerVer":"5.9.31",
      "apiVer":"3",
      "configured":true,
      "type":1,
      "supportApp":true,
      "omadacId":"***********",
      "registeredRoot":true
   }
}
```

Traceback :
```
Traceback (most recent call last):
  File "C:\Users\***\PycharmProjects\OmadaClient\main.py", line 51, in <module>
    asyncio.run(connect_omada())
  File "C:\Users\***\AppData\Local\Programs\Python\Python311\Lib\asyncio\runners.py", line 190, in run
    return runner.run(main)
           ^^^^^^^^^^^^^^^^
  File "C:\Users\***\AppData\Local\Programs\Python\Python311\Lib\asyncio\runners.py", line 118, in run
    return self._loop.run_until_complete(task)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\***\AppData\Local\Programs\Python\Python311\Lib\asyncio\base_events.py", line 650, in run_until_complete
    return future.result()
           ^^^^^^^^^^^^^^^
  File "C:\Users\***\PycharmProjects\OmadaClient\main.py", line 36, in connect_omada
    print(f'Port {port.port}: name={port.name}, link={port.port_status.link_status.name}')
                                                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\***\PycharmProjects\OmadaClient\venv\Lib\site-packages\tplink_omada_client\devices.py", line 159, in link_status
    return LinkStatus(self._data["linkStatus"])
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Olivier\AppData\Local\Programs\Python\Python311\Lib\enum.py", line 695, in __call__
    return cls.__new__(cls, value)
           ^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Olivier\AppData\Local\Programs\Python\Python311\Lib\enum.py", line 1111, in __new__
    raise ve_exc
ValueError: 1 is not a valid LinkStatus
```

**Comment by MarkGodwin** [comment]
This is fixed in v1.2.5 - thanks so much for your help!

**Comment by odouville** [comment]
Thanks for the merge!
